src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -84,11 +105,52 @@ public virtual async Task<ScaleStatusResult> GetScaleStatusAsync(ScaleStatusCont
                 // this can happen if the host is offline
             }
 
-            var vote = GetAggregateScaleVote(votes, context, _logger);
+            List<int> targetScaleVotes = new List<int>();
+
+            if (targetScalersToProcess.Any())
```

Comment:
Let's break these two blocks out into private instance helpers GetScaleMonitorsResult/GetTargetScalersResult. That'll keep the flow of this method clean.